lib/merit/models/active_record/merit/sash.rb

```diff
@@ -25,5 +25,9 @@ def score_points(options = {})
       end
       scope
     end
+
+    def badge_count
+      badges_sashes.count
+    end
```

Comment:
There's already module `Merit::Base::Sash.badges` method defined. The model is included in `Merit::Sash`, and also goes through `badges_sashes` in its implementation. Calling existing `badges.count` seems better than adding a new (ActiveRecord) specific version. What are your thoughts?